epidose/device/beacon_rx_unlinkable_d.py

```diff
@@ -72,6 +75,11 @@ def process_packet(socket):
     if packet[13:25] != BLE_PACKET:
         return
 
+    # Print sender's bdaddr
+    bdaddr = packet[7:13]
+    bdaddr = bdaddr[::-1].hex().upper()
+    logger.info(f"Received packet from {':'.join(bdaddr[i:i+2] for i in range(0, len(bdaddr), 2))}")
```

Comment:
Make this `.debug`; we don't want it in production.

Reply:
OK